.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,31 +1,11 @@
 <!-- Please refer to our contributing documentation for any questions on submitting a pull request, or let us know here if you need any help: https://ionicframework.com/docs/building/contributing -->
 
-## Pull request checklist
-
-Please check if your PR fulfills the following requirements:
-- [ ] Tests for the changes have been added (for bug fixes / features)
-- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
-  - Some docs updates need to be made in the `ionic-docs` repo, in a separate PR. See the [contributing guide](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#modifying-documentation) for details.
-- [ ] Build (`npm run build`) was run locally and any changes were pushed
-- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures
-
-
-## Pull request type
+<!-- Some docs updates need to be made in the `ionic-docs` repo, in a separate PR. See the https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#modifying-documentation for details. -->
 
 <!-- Please do not submit updates to dependencies unless it fixes an issue. --> 
 
 <!-- Please try to limit your pull request to one type, submit multiple pull requests if needed. --> 
```

Comment:
```suggestion
<!-- Please try to limit your pull request to one type (bugfix, feature, etc). Submit multiple pull requests if needed. --> 
```
(Optional) This context used to be included via the checklist; I think it's useful to be explicit.